src/commands/chatters/artemis.ts

```diff
@@ -3,51 +3,56 @@ import { Discord, SimpleCommand, SimpleCommandMessage, SimpleCommandOption, Simp
-    '12. Tolkien did not receive the total amount he was owed for the LotR Trilogy.',
-    "13. Filming the destruction of Sauron's tower was difficult.",
-    '14. The LotR trilogy could not fully make use of CGI for its effects. ',
-    '15. Tolkien originally did not want to name the third book The Return of the King.',
-    "16. Tolkien and his wife's gravestones are engraved with two names from LotR.",
-    '17. The University of California offers a LotR dorm for its freshmen.',
-    '18. Frodo is depicted as a clumsy character in the films.',
-    '19. Christopher Lee has always wanted to play the role of Gandalf. ',
-    '20. Nicolas Cage was originally supposed to play the role of Aragorn.',
-    '21. The fight scenes were choreographed by an Olympic fencer.',
-    '22. Viggo Mortensen did his stunts without a double.',
-    '23. Orlando Bloom broke his ribs during filming. ',
-    '24. John Rhy-Davies is the tallest actor in the fellowship. ',
-    "25. Peter Jackson's daughter makes several cameos throughout the films.",
-    '26. Denethor could not light the pyre on fire.',
-    '27. Green ping pong balls were used as stand-ins. ',
-    '28. The Queen of Denmark illustrated the Danish edition of LotR.',
-    '29. Nazgul screeches were made with plastic cups. ',
-    '30. The Gollum Effect was loosely based on the withdrawal symptoms of heroin addicts.',
-    "31. The filming of the battle of Helm's Deep took forever to finish.",
-    '32. Gollums were originally kind creatures. ',
-    "33. The Dead Marshes were said to be based on Tolkien's experience in WWI.",
-    '34. Sean Astin was knocked unconscious during filming.',
-    '35. The LotR franchise has won every award it was nominated for.',
-    'https://www.youtube.com/watch?v=xqeYKf8tdsU', // Bonus answer
+    'Viggo Mortensen (Aragorn) bought 3 horses that were used in the films.',
+    "The son of Tolkien does not like the LotR movies. (He's wrong)",
+    'Sean Bean climbed the mountains of New Zealand in full costume daily to get to the set, due to being too scared to ride in a helicopter.',
+    'The LotR movies cost less to make compared to the Hobbit films.',
+    'Billy Boyd (Pippin) named his son Faramir.',
+    'The Beatles tried to make their own LotR movie.',
+    'During filming Christopher Lee (Saruman) corrected Peter Jackson on the appropriate sound a man would make when he was stabbed in the back, stating, “Do you know what it actually sounds like when someone is stabbed? Because I do.”',
+    'Merry was originally named Marmaduke Brandybuck.',
+    "Billy Boyd (Pippin) wrote the music for the song “Edge of Night” which he sings in Denethor's halls. ",
+    'The theatrical edition of the movies is 9h18m. The extended edition is 11h22m.',
+    "All of the mountains on Saturn's largest moon are named after the mountains from LotR.",
+    'John Noble (Denethor) ate sloppily because he was starved for a month before filming to “make the greed more real”',
```

Comment:
```suggestion
    'John Noble (Denethor) ate sloppily because he was starved for a month before filming. “It was important to make the greed more real,” he explained.',
```